src/osip-monitoring-view/src/main/java/edu/kit/pse/osip/monitoring/view/dashboard/UIOutputStream.java

```diff
@@ -0,0 +1,207 @@
+package edu.kit.pse.osip.monitoring.view.dashboard;
+
+import java.io.PrintStream;
+import java.util.Locale;
+
+/**
+ * This class can be used to print messages of the default output and error output stream to a GUI console.
```

Comment:
https://github.com/ByteHamster/PSE/blob/master/docs/Design.tex#L198

Von den Exceptions hieß es da nichts. Dachte, dass wir nur die Alarme wollen (Exceptions weiter einfach nach stdout). Keine Ahnung, mal in der Gruppe absprechen